content/billing/scout-billing.md

```diff
@@ -6,71 +6,63 @@ keywords: payments, billing, subscription, scout
-8. Update the quantity of repositories you want to use with Docker Scout. See [Docker Scout subscriptions](../subscription/scout-details.md) to learn more about the number of repositories available for each plan.
-9. Select annual or monthly billing.
+In this section, learn how to buy and manage a Docker Scout Team subscription for your personal account or for an organization. If you're an organization with multiple teams, a Docker Scout Business plan may be better. To buy Docker Scout Business, [contact sales](https://www.docker.com/products/docker-scout/#contact-sales).
+
+> **Note**
+>
+> You need to be an owner to manage billing for an organization.
+>
+
+## Buy Docker Scout Team
+
+1. Sign in to [Docker Scout](https://scout.docker.com/) with your Docker ID.
+2. Select your personal account or organization namespace from the drop-down in the navigation menu on the left side of the screen.
+3. Select **Settings**.
+4. Select **Billing**.
+5. The default plan is Docker Scout Free. Next to the plan name, select **Change plan**.
+6. Select **Purchase Scout Team**.
+7. Choose annual or monthly billing cycle, then select the number of Scout-enabled repositories you want from the repository quantity drop-down. You can select groups of five.
+8. Select **Continue to payment**.
+9. This redirects you to the payment processing page. Enter your email if this field isn't pre-populated. Then, enter your payment information.
 10. Select **Subscribe**.
-11. Subscribing redirects you back to the billing page in Docker Hub, where you can find your active Docker Scout plan.
 
-Once your purchase is complete, you receive a confirmation email and a copy of your invoice.
+Subscribing redirects you back to the billing page in Docker Scout, where you can find your active Docker Scout plan. Once your purchase is complete, you receive a confirmation email and a copy of your invoice.
 
 ## Manage your subscription
 
-To access your subscription details, go to the **Billing** section for your personal account or organization that's subscribed. Select **Payment methods and billing history** to go to the billing details portal. In the billing details portal, you can find your renewal date, invoice history, and payment methods.
+To access your subscription details, sign in to [Docker Scout](https://scout.docker.com/) with your Docker ID. Select **Settings**, then **Billing** for your personal account or organization that's subscribed.
+
+Here you can find:
+
+- The type of billing cycle you have, annual or monthly
+- How many repositories are Scout-enabled
+- Your subscription renewal date
 
-Renewals charge to the original credit card used to buy Docker Scout Team. You can manage your payment methods in the billing details portal.
+Renewals charge to the original credit card used to buy Docker Scout Team. To update your payment method, see [Add or update a payment method](/billing/payment-method/).
```

Comment:
**discussion:** Scout will embed the "payment method and billing history" link on its own pages very soon 🙏

Reply:
I can update this section in a separate PR once that is live 👍

Reply:
This is now live 
<img width="1099" alt="Screenshot 2024-03-01 at 09 27 28" src="https://github.com/docker/docs/assets/13132227/f0ac9621-51ec-47db-8b5a-251dccbb8231">

Reply:
Updated this here!